.github/workflows/build_apps.yml

```diff
@@ -0,0 +1,41 @@
+name: Build_apps
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
```

Comment:
This is tricky. We would need:

1. Binaries should be built in PRs, to verify that the build wouldn't fail when the changes are merged into `master`.
1. Binaries should be (built again and) released only when we push a version tag. ([Example](https://github.com/bittner/pyclean/blob/master/.github/workflows/packaging.yml#L3-L6))

What we probably don't want is a new release each time we push or merge something into `master`.

See also: [Related documentation](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags)